Header header

# DCDC-USB connected
bool ready

# Input voltage
float64 input_voltage

#Output voltage
float64 output_voltage

